src/server/rest/v1/router/api/TransactionRouter.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { ServerAction, ServerRoute } from '../../../../../types/Server';
+import express, { NextFunction, Request, Response } from 'express';
+
+import RouterUtils from '../RouterUtils';
+import TransactionService from '../../service/TransactionService';
+
+export default class TransactionRouter {
+  private router: express.Router;
+
+  public constructor() {
+    this.router = express.Router();
+  }
+
+  public buildRoutes(): express.Router {
+    this.buildRouteCompletedTransactions();
+    return this.router;
+  }
+
+  protected buildRouteCompletedTransactions(): void {
+    this.router.get(`/${ServerRoute.REST_TRANSACTIONS}`, async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Just for my understanding, that endpoint is supposed to get all the transactions completed on one tenant?

Reply:
> Just for my understanding, that endpoint is supposed to give all the transactions completed on one tenant?
> No time frame filtering, no pagination?

Yes, all the completed transactions and filtering, pagination and projection are passed in the query string.
If not passed default are set.